doc/htmldoc/hpc/mpi_processes.rst

```diff
@@ -0,0 +1,79 @@
+.. _mpi_process:
+
+MPI process
+===========
+
+
+An MPI process is process of a computer program that is being executed by one or many threads and employs a Message Passing Interface (MPI) to communicate its data with the other processes.
+
+We typically use the standard Message Passing Interface (MPI) to instruct how processes work in parallel (See e.g.,
+`OpenMPI <https://www.open-mpi.org/>`_).
+
+The number of MPI processes on each each node is related to the number of :ref:`threads <threads>` you want for each MPI process.
+Multiplied together, the values should equal the total number of cores in a node. (The number of cores varies depending on what system you are using).
```

Comment:
This is not true, one can have as many as MPI processes and threads one wants, the problem would then be whether the node can run everything in parallel or not.
Along with the previous sentence, these statements confuse the usage of MPI processes along with their threads and the distribution of these to the available computing resources.